api/cpp/include/slint.h

```diff
@@ -846,6 +848,16 @@ class Repeater
         return { &C::static_vtable, const_cast<C *>(&(**x.ptr)) };
     }
 
+    void component_at(int i, vtable::VWeak<private_api::ComponentVTable> *result) const
```

Comment:
I suggest to return the VWeak by value here. After all this is called by generated code, right?

It would also be consistent with the Repeater in Rust.

Reply:
Good point, done.